lm_eval/tasks/marc_ja.py

```diff
@@ -83,8 +83,64 @@ def construct_requests(self, doc, ctx):
 
         return lls
 
+
+
+class MARCJaWithJAAlpacaPrompt(MARCJaWithFintanPrompt):
+    """
+    This prompt format was inspired by the below data in fujiki/japanese_alpaca_data. 
+    ```
+    {
+        'instruction': '以下のテキストを、ポジティブまたはネガティブの感情クラスのいずれかに分類してください。', 
+        'input': '製品が遅すぎて使い勝手が悪かったので、あまり好きではありませんでした。', 
+        'output': 'ネガティブ。'
```

Comment:
@fujiki-saij lm-eval-harness in this task computes scores across the class labels rather than comparison between labels and *generated* texts. https://github.com/Stability-AI/lm-evaluation-harness/blob/3b25c523ae68784127fb9fd0caa09de6053fc054/lm_eval/tasks/marc_ja.py#L105
So, I believe it doesn't affect.